### ISSUE TITLE
"Attempted to dereference garbage pointer" crash in KSCrash 1.15.3

### ISSUE THREAD

**Issue by rivera-ernesto** [kstenerud/KSCrash#209]
```OS Version: iOS 10.2 (14C92)
Report Version: 104

Exception Type: EXC_BAD_ACCESS (SIGBUS)
Crashed Thread: 2

Application Specific Information:
Attempted to dereference garbage pointer 0x16e023180.

...

Thread 2 name: 
Thread 2 Crashed:
0   KSCrash                         0x5f35fc            ksthread_getQueueName
1   KSCrash                         0x5f35e0            ksthread_getQueueName
2   KSCrash                         0x5cb924            ksccd_init
3   libsystem_pthread.dylib         0x101a97850         _pthread_body
4   libsystem_pthread.dylib         0x101a97760         _pthread_start```

**Comment by rivera-ernesto** [comment]
Same here https://sentry.io/share/issue/3131313230352e323331333036383736/

**Comment by edenman** [comment]
Same issue here, but with a different caller of ksthread_getQueueName.  And with line numbers:

module|address|method name|file-and-line-number
---------|-------------|-------------------------|------------------
KSCrash|0x100d9f20c|ksthread_getQueueName|KSThread.c:82
KSCrash|0x100d9f1f0|ksthread_getQueueName|KSThread.c:75
KSCrash|0x100d77cbc|monitorCachedData|KSCrashCachedData.c:139
KSCrash|0x100d77cbc|updateThreadList|KSCrashCachedData.c:84
libsystem_pthread|0x18092d850|__pthread_body
libsystem_pthread|0x18092d760|__pthread_start

**Comment by edenman** [comment]
fwiw, #207 was filed first and seems to be the same issue.

**Comment by edenman** [comment]
(this can also be closed since #207 was fixed)
